central/pruning/pruning.go

```diff
@@ -430,10 +438,18 @@ func (g *garbageCollectorImpl) removeOrphanedProcessBaselines(deployments set.Fr
 // removeOrphanedPLOPs: cleans up ProcessListeningOnPort objects that are expired
 // or have a PodUid and belong to a deployment or pod that does not exist.
 func (g *garbageCollectorImpl) removeOrphanedPLOPs() {
-	prunedCount := postgres.PruneOrphanedPLOPs(pruningCtx, g.postgres, orphanWindow)
+	prunedCount := g.plops.PruneOrphanedPLOPs(pruningCtx, orphanWindow)
 
 	log.Infof("[PLOP pruning] Found %d orphaned process listening on port objects",
 		prunedCount)
+
+	prunedCount, err := g.plops.RemovePLOPsWithoutProcessIndicatorOrProcessInfo(pruningCtx)
+
+	log.Infof("[PLOP pruning] Pruning of %d orphaned PLOPs with no matching process indicator or process information complete", prunedCount)
```

Comment:
Check the error before the log and if theres an error then don't print?

Reply:
I switched the order.